src/commands/user/export-mapping.ts

```diff
@@ -0,0 +1,42 @@
+import fs from 'fs'
+import path from 'path'
+
+import { flags } from '@oclif/command'
+import { Kommand } from '../../common'
+import { kuzzleFlags } from '../../support/kuzzle'
+
+export default class UserExportMapping extends Kommand {
+  static description = 'Exports user mapping to JSON.'
```

Comment:
Mappings should always be used in the plural form (I know the API route is `updateMapping` but that's an error..)
```suggestion
  static description = 'Exports user mappings to JSON.'
```

Reply:
`userS mappings` ?